### ISSUE TITLE
Missing areas

### ISSUE THREAD

**Issue by jessjaco** [digitalearthpacific/dep-grid#2]
Such as the selected atolls here, part of French Polynesia
![image](https://github.com/digitalearthpacific/dep-grid/assets/1250693/2e24d9c6-fc67-4397-bd84-343e8de6c27d)

**Comment by alexgleith** [comment]
Are these still missing?

**Comment by jessjaco** [comment]
No, I think the core issue is the use of the land boundaries which exist here vs gadm. I think gadm/"padm" is better, would be nice if @sopac confirmed as well
